public/config.js

```diff
@@ -0,0 +1,5 @@
+var config = {
+  apiKey: "AIzaSyBkyP0zcL3AvWL3m8ZD67gBtVpfWCACI2E",
```

Comment:
yes in theory, but theres no way to I can't figure out how to configure it with firebase hosting...